google-cloud-tools-plugin/resources/META-INF/plugin.xml

```diff
@@ -33,14 +33,43 @@
   <!-- "idea-version since-build" set by gradle-intellij-plugin -->
 
   <change-notes>
-    <![CDATA[<html>
+    <![CDATA[
+    <html>
       <p>
       IMPORTANT:
       This plugin requires the use of Cloud SDK v 133.0.0 for correct execution of the local
       development server with the latest Java 8 SDK. Please run 'gcloud components update' from your
       shell to ensure you have the latest Cloud SDK release.
       </p>
 
+      <h2>[17.2.1]</h2>
+
+      <p>Happy new year Cloud Tools for IntelliJ users! This year's first release is primarily a maintenance
+        release.  If you are having authentication problems using Cloud Source Repositories and our
+        plugin, check out <a href="https://github.com/GoogleCloudPlatform/google-cloud-intellij/issues/1174">this possible solution</a>.</p>
```

Comment:
you could link to the user's comment where he says he gets unblocked:

https://github.com/GoogleCloudPlatform/google-cloud-intellij/issues/1174#issuecomment-275440330